### DISCUSSION TITLE
Should MultiPolygon Buildings Be Allowed?

### DISCUSSION THREAD

**Discussion by youngpm** [OvertureMaps/data#287] [Discussion]
I was looking at the `building` data in the September release over Manhattan, and there's 5 buildings of MultiPolygon type:
```
┌───────────────────────────┬──────────────┐
│ st_geometrytype(geometry) │ count_star() │
│       geometry_type       │    int64     │
├───────────────────────────┼──────────────┤
│ POLYGON                   │        45754 │
│ MULTIPOLYGON              │            5 │
└───────────────────────────┴──────────────┘
```

These all come from OSM relations:
```
┌──────────────────────────────────┬────────────────────────┐
│                id                │ (sources[1]).record_id │
│             varchar              │        varchar         │
├──────────────────────────────────┼────────────────────────┤
│ 08b2a1072c775fff020073aa0caa2d81 │ r16951153@1            │
│ 08b2a1072dc94fff02002e137a88853c │ r17233745@1            │
│ 08b2a1072c0f5fff0200eb11146127ec │ r7597658@1             │
│ 08b2a10725352fff02002e76470f1a6f │ r17015722@1            │
│ 08b2a100ae743fff0200d4c4749618ad │ r9219294@3             │
└──────────────────────────────────┴────────────────────────┘
```
[16951153](http://osm.org/relation/16951153)
[17233745](http://osm.org/relation/17233745)
[7597658](http://osm.org/relation/7597658)
[17015722](http://osm.org/relation/17015722)
[9219294](http://osm.org/relation/9219294)

The [OSM tagging wiki](https://wiki.openstreetmap.org/wiki/Key:building) doesn't seem to offer guidance (or I missed it) but MultiPolygon buildings don't make much sense to me; would it make sense for Overture to be more restrictive here?

**Comment by Azbesciak** [comment]
I would say even more - there is `building_part`, right? I thought that was the place for such cases.
